packages/kit/src/runtime/app/images.js

```diff
@@ -0,0 +1,47 @@
+import { DEV } from 'esm-env';
+import { sizes, loader, domains } from '__sveltekit/images';
+
+/**
+ * @param {string} src
+ * @param {any} [options]
+ * @returns {{ src: string, srcset?: string }}
+ */
+export function getImage(src, options) {
+	if (DEV) {
+		if (!matches_domain(src)) {
+			console.warn(
+				`$app/images: Image src '${src}' does not match any of the allowed domains and will therefore not be optimized.`
+			);
+		}
+		return { srcset: src, src };
+	}
+
+	if (!matches_domain(src)) {
+		return { src };
+	}
+
+	const srcset = sizes
```

Comment:
I don't really understand how this is supposed to be used. If the `img` tag doesn't have the `sizes` attribute set there's no point in providing a `srcset`. `sizes` is very rarely used, so it seems like generating this will mostly be wasted.

Reply:
I went back and looked at your old PR as saw that the `Image` component was setting `sizes` automatically there: https://github.com/sveltejs/kit/blob/6bca66026e3029f77c1a78b4c67b934a09e7547c/packages/image/src/Image.svelte

I'm not sure if you intended to do that here as well and just missed copying it over or if you meant to do it differently here?